<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelAcquisitionWindow</class>
 <widget class="QMainWindow" name="ModelAcquisitionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>614</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>577</width>
    <height>614</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>577</width>
    <height>614</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>3D View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ntk::ImageWidget" name="colorView" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>388</width>
      <height>272</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetCamera">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveMeshButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save </string>
    </property>
   </widget>
   <widget class="ntk::MeshViewer" name="mesh_view" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>491</width>
      <height>321</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>10000</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="ntk::ImageWidget" name="depthView" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>10</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAsMeshButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save  as</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grabButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Grab</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>577</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMeshgrabber">
    <property name="title">
     <string>Meshgrabber</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMeshgrabber"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string>dsscdsc</string>
   </property>
  </widget>
  <action name="action_Grab_and_Move">
   <property name="text">
    <string>Grab and Move</string>
   </property>
  </action>
  <action name="action_Grab_Sequence">
   <property name="text">
    <string>Grab Sequence</string>
   </property>
  </action>
  <action name="actionModel_and_Move">
   <property name="text">
    <string>Model and Move</string>
   </property>
  </action>
  <action name="actionModel_Sequence">
   <property name="text">
    <string>Model Sequence</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ntk::ImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">ntk/gui/image_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ntk::MeshViewer</class>
   <extends>QWidget</extends>
   <header location="global">ntk/mesh/mesh_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
